staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/helpers.go

```diff
@@ -65,7 +65,7 @@ func NestedString(obj map[string]interface{}, fields ...string) (string, bool, e
 	}
 	s, ok := val.(string)
 	if !ok {
-		return "", false, fmt.Errorf("%v is of the type %T, expected string", val, val)
+		return "", true, fmt.Errorf("%v is of the type %T, expected string", val, val)
```

Comment:
Hrm, not sure why this is showing up here. Was updated in https://github.com/kubernetes/kubernetes/pull/56563. Will remove it from here.